packages/types/src/primitive/sszTypes.ts

```diff
@@ -65,3 +65,6 @@ export const BLSSignature = Bytes96;
 export const Domain = Bytes32;
 export const ParticipationFlags = new UintNumberType(1, {setBitwiseOR: true});
 export const ExecutionAddress = new ExecutionAddressType();
+export const ColumnIndex = UintNum64;
+export const CustodyIndex = UintNum64;
+export const RowIndex = UintNum64;
```

Comment:
`ColumnIndex` and `CustodyIndex` are unique and requisite.  `RowIndex` can go though 

```suggestion
export const ColumnIndex = UintNum64;
export const CustodyIndex = UintNum64;
```